client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1807,6 +1807,51 @@ Undocumented.prototype.submitSupportForumsTopic = function( subject, message, fn
 };
 
 /**
+ * Get the available export configuration settings for a site
+ *
+ * @param {int}       siteId            The site ID
+ * @param {Object}    advancedSettings  Advanced export configuration
+ * @param {Function}  fn                The callback function
+ * @api public
+ */
+Undocumented.prototype.getExportSettings = function( siteId, fn ) {
+	this.wpcom.req.get( {
+		apiVersion: '1.1',
+		path: `/sites/${ siteId }/exports/settings`
+	}, fn );
```

Comment:
if you return these calls, then your code can take advantage of that by inspecting the request object.

additionally, if you return them and don't use a callback, the `wpcom.js` library will return a Promise that resolves on the successful response from the API and rejects on error

```js
Undocumented.prototype.getExportSettings = function( siteId ) {
	return this.wpcom.req.get( {
		apiVersion: '1.1',
		path: `/sites/${ siteId }/exports/settings`
	} );
}

//~~~ another file ~~~~
wpcom.getExportSettings( siteId )
	.then( receiveExportSettings )
	.catch( apiFailure );
```